opm/simulators/flow/GenericThresholdPressure.hpp

```diff
@@ -23,15 +23,14 @@
 /*!
  * \file
  *
- * \copydoc Opm::EclThresholdPressure
+ * \copydoc Opm::ThresholdPressure
```

Comment:
Same here. But originally Generic is missing also. I do not have a strong opinion.

Reply:
Same as above.